packages/path_provider/path_provider_windows/example/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: path_provider_example
+description: Demonstrates how to use the path_provider plugin.
+
+dependencies:
+  flutter:
+    sdk: flutter
+  path_provider: any
+  path_provider_windows:
```

Comment:
IIRC this needs to be done as an override, since it'll conflict with the path_provider version of the dependency.